src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from "react";
+import { RadioGroup as ChakraRadioGroup, Stack } from "@chakra-ui/react";
+
+import HelperErrorText from "../HelperErrorText/HelperErrorText";
+import generateUUID from "../../helpers/generateUUID";
+import { spacing } from "../../theme/foundations/spacing";
+
+export enum RadioGroupLayouts {
+  Vertical = "vertical",
+  Horizontal = "horizontal",
+}
+
+type DirectionMap = { [key: string]: "column" | "row" };
```

Comment:
These are the values for Chakra's `direction prop`. I think we should use "column"/"row" instead of "vertical"/"horizontal" to be consistent with Chakra's convention. If we're moving to other prop verb usage such as "isDisabled", "isErrored", then it make sense to update this.

@bigfishdesign13 let me know what you think and if you agree, the spec doc should be updated.

Reply:
Yes, again for better alignment with Chakra, let's move to `column` and `row`.  On a related note, both `Card` and `SkeletonLoader` use `horizontal` and `vertical` values for their `layout` props.  We'll need to discuss whether this same verbiage change would be applicable there.

Reply:
I think it should be applicable everywhere just like we decided on the `isDisabled`, `isErrored`, etc. But that can be a refactoring ticket or whenever other components are updated.

Reply:
I created new tickets to update the `layout` prop in `Card` and `SkeletonLoader`.